Vagrantfile

```diff
@@ -68,8 +70,8 @@ def chef_defaults(chef, name, environment = 'splunk_server')
 end
 
 Vagrant.configure('2') do |config|
-  config.vm.box = 'bento/centos-6.7'
-  config.ohai.primary_nic = 'eth1'
+  config.vm.box = 'bento/centos-7.2'
```

Comment:
Talked to Alec offline. I'm seeing an error https://gist.github.com/nmattam/4ef676fd38f77f8710e1126e5a333084 and when I revert this change, it works fine.